app/classes/Langchecker/GetTextManager.php

```diff
@@ -74,16 +74,15 @@ public static function importLocamotion($locale_data, $current_filename, $curren
           'strings'  => [],
         ];
 
-        Utils::logger("== {$current_locale} ==");
-
         $local_import = $locamotion_repo != '' ? true : false;
 
-        /* Some locales need to me mapped because our systems and Pootle use
-         * different locale codes.
-         *
-         * Array format: local code => pootle code
-         *
-         * Use dashes and not underscores for Pootle, e.g. gn-PY (not gn_PY).
+        /*
+            Some locales need to me mapped because our systems and Pootle use
+            different locale codes.
+
+            Array format: Langchecker code => Pootle code
+
+            Use dashes and not underscores for Pootle, e.g. gn-PY instead of gn_PY.
          */
```

Comment:
I see php.net has a [page with an example](http://php.net/manual/en/language.basic-syntax.comments.php) like Pascal's style, let's stick to that.